contrib/chatops/actions/format_execution_result.py

```diff
@@ -42,3 +43,12 @@ def run(self, execution):
                     template = alias.result['format']
 
         return self.jinja.from_string(template).render(context)
+
+    def _get_execution(self, execution_id):
+        if not execution_id:
+            raise ValueError('Invalid execution_id provided.')
+        execution = self.client.liveactions.get_by_id(id=execution_id)
```

Comment:
So is it retrieving an execution or live action?

This stuff still is confusing - either client manager name is incorrect or the method, argument, etc. here is incorrect.